subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/JdksBlockForToolchainManagement.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.jvm.toolchain;
+
+import org.gradle.api.Action;
+import org.gradle.api.Incubating;
+
+/**
+ * Dynamic extension added to <code>ToolchainManagementSpec</code> at runtime, by the
+ * <code>jvm-toolchains</code> plugin. Provides a <code>jdks</code> block to configure
+ * the ordered list of <code>JavaToolchainRepository</code> implementations to use
+ * for Java toolchain provisioning.
+ *
+ * @since 7.6
+ */
+@Incubating
+public interface JdksBlockForToolchainManagement {
+
+    // TODO (#21082): update user manual
+
+    /**
+     * Returns the handler for the ordered list of configured <code>JavaToolchainRepository</code> implementations.
+     */
+    JavaToolchainRepositoryResolverHandler getResolvers();
```

Comment:
I would use `jdkResolvers` as the name, adapting this getter and the method below